organizational-structure-overview.md

```diff
@@ -0,0 +1,86 @@
+- The **Technical Advisory Council** (TAC) is responsible for the general success of all Technical Initiatives (defined below). TAC meetings are generally open to the public, with the exception of special sessions. 
+  - TAC membership is limited to seven (7) seats, with alternating elections every year. Contributors to Technical Initiatves may run, and each contributor may vote once in TAC elections. 
+- **Technical Initiatives** are groups formed in support of the overall mission of the OpenSSF. Technical Initiatives include Working Groups (WG), Special Interest Groups (SIG), Projects, and services governed by their own Charters with various objectives, reporting relationships, and funding models described below.
+   - All Technical Initiatives are open groups focused on a technical objective, with transparent proceedings that are open to the public, and vary in their objectives.
+   - Technical Initiatives usually manage one or more GitHub repos of their own, may have separate meetings.
+- A **Project** is a Technical Initiative focused on the development and ongoing support of open source licensed software (source code) and its supporting artifacts (technical documentation, etc.).
+- A **Working Group** (WG) focuses on developing other types of deliverables than software such as guides, specifications, and educational material, or conducting initiatives such as an education outreach. While a WG does not produce open source software as a primary artifact it may launch Projects that do so. A WG may also launch Special Interest Groups (SIG) to perform specific tasks. WGs often include some open source code, or use licensed software, in fulfilment of their Charter.
+- **Special Interest Groups** (SIG) are under the direct governance of their reporting WGs and are bound to achieving a very specific goal. 
+- A **Technical Deliverable** can be an open source licensed software and its supporting artifacts, a specification, a technical guide. This is a technical artefact produced by a Technical Initiative.
+- A **Service** is a Technical Initiative in which software is either built or acquired to support or automate OSSF transactions.
+
+The following table describes the main types of groups and their characteristics.
+
+| Initiative | Expected lifespan | Primary output| Reporting relationship |	Economic model
+|------------|-------------------|---------------|------------------------|---------------
+| Working Group (WG) | unbounded | not software | to the TAC | normative
+| WG w/ SIF | unbounded | not software | to the TAC | special fund
+| Project |	unbounded | software | either TAC or WG | normative
+| Project w/ SIF | unbounded | software | to the TAC | special fund
+| Special Interest Group (WG) | bounded | not software | to a WG | normative
+
+SIF = Specific Initiative Fund
+
+### TODO
+
+* define **SIF** 
```

Comment:
Should we punt this TODO, or is it in scope for the merge?

Reply:
My thinking is that we should punt because it will take longer to get this one sorted and I'd hate to block the rest of these items, but pause new SIF creation until that is sorted out.

Reply:
Agreed, let's punt this. This PR is getting large - the more we can do things in smaller increments, the better. Jory & others have been working on defining that separately.

Reply:
+1 to punting this.